backend/de.metas.externalsystem/src/main/java/de/metas/externalsystem/ExternalSystemParentConfig.java

```diff
@@ -36,18 +36,21 @@
 	ExternalSystemType type;
 	String name;
 	IExternalSystemChildConfig childConfig;
+	Boolean isActive;
```

Comment:
why not primitive `boolean`?

btw, if u need also the `null`.... better use OptionalBoolean

Reply:
We want to be sure that isActive is set every time to the 'actual' value and not set to false by default (default value of primitive boolean) if someone forgets to set it while using builder.